File: keychain-sample-chat-dot-net/KeychainChat/Services/DirectoryEntry.cs
namespace KeychainChat.Services
{
    public class DirectoryEntry
    {
        public string name { get; set; }
        public string encr_txid { get; set; }
        public string encr_vout { get; set; }
        public string sign_txid { get; set; }
        public string sign_vout { get; set; }
    }
}

File: keychain-sample-chat-iphone/KeychainChat/Models/DirectoryEntry.swift
//
//  DirectoryEntry.swift
//  
//
//  Created by Robert Ellis on 2021/12/07.
//

import Foundation

public struct DirectoryEntry: Codable {
    public let name: String
    
    public let encr_txid: String

    public let encr_vout: Int

    public let sign_txid: String

    public let sign_vout: Int
}
